website/docs/docs/deploy/job-notifications.md

```diff
@@ -4,7 +4,13 @@ id: "job-notifications"
 description: "Set up notifications in dbt Cloud to receive email or Slack alerts about job run status."
 ---
 
-Set up notifications in dbt Cloud to receive email or Slack alerts when a job run succeeds, fails, or is cancelled.
+
+Set up notifications in dbt Cloud to receive email or Slack alerts about the status of a job run. You can choose to be notified by one or more of the following job run statuses: 
+
+- **Succeeds** option &mdash; A job run completed successfully.
+- **Warns** option &mdash; A job run encountered warnings from [tests](/docs/build/data-tests) or [source freshness](/docs/deploy/source-freshness) checks (if applicable).
```

Comment:
Is it only those two scenarios? Because we can warn about a lot of things. If it is, it might be beneficial to explicitly note that other `warning` scenarios aren't covered

Reply:
Yes, the only scenarios covered are warnings you explicitly set. We don't warn on things like depreciating functionality, or we would be sending you a warning with every single run, and these notifications would not be used because they would be so noisy. 

This should be the expected behavior from most customers, but we can call it out explicitly if we want!